src/models/backend/transfers.rs

```diff
@@ -39,13 +40,29 @@ pub struct Erc20Transfer {
     pub block_number: u64,
     pub transaction_hash: String,
     pub to: String,
+    #[serde(deserialize_with = "default_if_null")]
```

Comment:
Should we do the same for tokenId in an erc721 transfer?

Reply:
I am not sure if `tokenId = 0` has a special meaning. I checked the standard quickly and seems to be ok. If you can think of a situation where `tokenId = 0` may be misunderstood let me know.

Reply:
it would be similarly confusing to a user as a wrong value amount, imo